topics/transcriptomics/tutorials/scrna-plant/tutorial.md

```diff
@@ -327,7 +327,7 @@ Dimension reduction is the art of reducing a high dimensional dataset into a low
 
 > ### {% icon tip %} Tip: Dimension Reduction
 >
-> You can learn more about dimension reduction by consulting the [Introduction to scRNA-seq video at slide XXXX]().
+> You can learn more about dimension reduction by consulting the [An introduction to scRNA-seq data analysis]({% link videos/#video-transcriptomics-scrna-intro %}) at time point 13:46.
```

Comment:
let's see if we can add a syntax for linking to timepoints in the future good to know this is a usecase!

Reply:
Btw, Is there a good way I can link videos?

Reply:
no :( this is best currently.

I've filed #2465, maybe we can do something better in the near future. There are so many possible ways to include it (e.g. 'mini' player on the tutorial page, etc.) we haven't scratched the surface yet. just need time to do these integrations